src/graphql/schemas/food.graphql

```diff
@@ -0,0 +1,7 @@
+extend type Query {
+  food(upc: String!): Food!
+}
+
+type Food {
```

Comment:
Should the data in this be broken down into `metaInfo` (category, brand owner, brand, UPC), `nutrientBreakdown` (energy, protein, fat, etc...), and `ingredients`?